server/utils/rating.util.js

```diff
@@ -0,0 +1,39 @@
+import db from '../models';
+
+const { Article } = db;
+
+class Rating {
+  /**
+   * Check the existence of an article
+   * @param {string} id
+   * @returns {boolean} aricleExists
+   */
+  static async doesArticleExist(id) {
+    return true;
+  }
+
+  /**
+   * Get the claps for an article
+   * @param {string} id
+   * @returns {number} claps
+   */
+  static async getArticleClaps(id) {
+    return 30;
+  }
+
+  /**
+   * Get the claps for an article
+   * @param {number} claps
+   * @returns {number} rating
+   */
+  static getArticleRating(claps) {
+    if (claps >= 0 && claps < 10) return 0;
```

Comment:
This if statement could be done better and neater

```
if(claps < 10) return 0;
if(claps < 20) return 1;
...
return 5;
```

Reply:
@paradisekelechi thanks for the review, we will refactor the code.